ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -117,6 +117,104 @@ tests:
           cpu: 300m
           memory: 500Mi
     workflow: generic-claim
+- as: e2e-aws-ocp-4.9-periodic
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 40m0s
+    version: "4.9"
```

Comment:
This is going to test the latest SRO (which is 4.10) with OCP 4.9, shouldnt this be with 4.10?

Reply:
There is no support for OCP 4.10 ATM.